prisma/schema.prisma

```diff
@@ -7,6 +7,28 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+enum FilterMode {
+  Global
+  User
+}
+
+enum FilterEntity {
+  Candidate
+}
+
+model Filter {
+  id      String       @id @default(cuid())
+  mode    FilterMode
+  entity  FilterEntity
+  params  String
+  default Boolean      @default(false)
+  userId  Int?
```

Comment:
I think with optional userId you don't need mode=User

Reply:
fixed